packages/cli/src/ui/components/Notifications.tsx

```diff
@@ -5,41 +5,77 @@
  */
 
 import { Box, Text, useIsScreenReaderEnabled } from 'ink';
+import { useEffect, useState } from 'react';
 import { useAppContext } from '../contexts/AppContext.js';
 import { useUIState } from '../contexts/UIStateContext.js';
 import { theme } from '../semantic-colors.js';
 import { StreamingState } from '../types.js';
 import { UpdateNotification } from './UpdateNotification.js';
 
 import { GEMINI_DIR } from '@google/gemini-cli-core';
-import { homedir } from 'node:os';
+
+import * as fs from 'node:fs';
+import os from 'node:os';
 import path from 'node:path';
 
-const settingsPath = path.join(homedir(), GEMINI_DIR, 'settings.json');
+const settingsPath = path.join(os.homedir(), GEMINI_DIR, 'settings.json');
+
+const screenReaderNudgeFilePath = path.join(
+  '/tmp',
+  GEMINI_DIR,
+  'hasSeenScreenReaderNudge',
```

Comment:
nit hasSeenScreenReaderNudge -> seen_screen_reader_nudge.json
or better yet pick file name you can use for all the tmp data that should not be in settings such as local_storage.json or something. Expect at least 
 'hasSeenIdeIntegrationNudge' should get moved here. Looking at the code I suspect hasSeenIdeIntegrationNudge is broken today as it is using the obsolete scheme name instead of `hasSeenIdeIntegrationNudge` instead of `ide.hasSeenNudge` unless we still happen to be doing enough migration of legacy schema names to make this work.